.github/workflows/cibuildwheels.yml

```
@@ -53,19 +53,29 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
-
-      - name: Install cibuildwheel
-        run: |
-          python -m pip install cibuildwheel
+          python-version: '3.11'
 
       - name: Set up QEMU
         if: ${{ matrix.arch == 'aarch64' }}
         uses: docker/setup-qemu-action@v3
 
-      - name: Build wheels
+      - name: Install Ninja
+        id: ninja
+        uses: turtlesec-no/get-ninja@main
+
+      - name: Install MSVC amd64
+        uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: amd64
+
+      - name: Update Clang
+        if: ${{ matrix.os == 'macos-latest' && matrix.arch == 'arm64' }}
         run: |
-          python -m cibuildwheel --output-dir wheelhouse
+          sudo xcode-select -s /Applications/Xcode_15.0.app/Contents/Developer
+          clang -v
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.20.0
```

```suggestion
        uses: pypa/cibuildwheel@v2.20
```
`cibuildwheel` intentionally do not provide `v2` aliases, only `vX.Y`